### ISSUE TITLE
RFE: improve message handling when getting HTTP/401 from AS3 

### ISSUE THREAD

**Issue by alonsocamaro** [F5Networks/k8s-bigip-ctlr#3351]
#### Title
RFE: improve message handling when getting HTTP/401 from AS3 

#### Description

During a customer PoC, I´ve found that there was a an authentication error in CIS, only the following was being sent:

```
$ oc logs k8s-bigip1
-ctlr-deployment-79d695b5ff-nsvjk
2024/03/22 12:30:16 [INFO] [INIT] Starting: Container Ingress Services - Version: 2.16.0, BuildInfo: azure-5835-34b43c48e8d0f41b0d6d0b80644027a9c8286dc3
2024/03/22 12:30:16 [INFO] ConfigWriter started: 0xc00074c750
2024/03/22 12:30:16 [WARNING] Creating GTM with default bigip credentials as GTM BIGIP Url or GTM BIGIP Username or GTM BIGIP Password is missing on CIS args.
2024/03/22 12:30:16 [INFO] Started config driver sub-process at pid: 18
2024/03/22 12:30:16 [INFO] [CORE] Registered BigIP Metrics
2024/03/22 12:30:16 [ERROR] [AS3] Error response from BIGIP with status code 401
2024/03/22 12:30:16 [ERROR] Error response from BIGIP with status code 401
2024/03/22 12:30:16 [INFO] [CCCL] ConfigWriter stopped: 0xc00074c750
```
where it is not clear at all what is going on, while if sending a request with POSTMAN the following is retrieved:

```
{
    "code": 401,
    "message": "Authentication failed: Password expired. Update password via /mgmt/shared/authz/users.",
    "restOperationId": 4501183,
    "kind": ":resterrorresponse"
}
```

where it is very clear what the problem is. I wonder if we could incorporate this in our logs.

Many thanks

        Ulises

**Comment by trinaths** [comment]
Created [CONTCNTR-4661] for internal tracking.

**Comment by mdditt2000** [comment]
Resolved in CIS 2.18 - https://clouddocs.f5.com/containers/latest/reference/release-notes.html
